doc/interop-test-descriptions.md

```diff
@@ -85,56 +83,96 @@ Procedure:
 
 Client asserts:
 * call was successful
-* response payload type is COMPRESSABLE
 * response payload body is 314159 bytes in size
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### large_compressed_unary
+### client_compressed_unary
 
-This test verifies compressed unary calls succeed in sending messages. It
-sends one unary request for every payload type, with and without requesting a
-compressed response from the server.
+This test verifies the client can compress unary messages. It sends an initial
+inconsistent request to verify whether the server supports the
+[CompressedRequest][] feature. If it does, it should catch the inconsistency and
+fail the call with an `INVALID_ARGUMENT` status. If the feature is supported, it
+proceeds with two unary calls, for compressed and uncompressed payloads.
 
-In all scenarios, whether compression was actually performed is determined by
-the compression bit in the response's message flags.
+Server features:
+* [UnaryCall][]
+* [CompressedRequest][]
+
+Procedure:
+ 1. Client calls UnaryCall with the feature probe, an **uncompressed** message:
+    ```
+    {
+      expect_compressed: false
+      response_size: 314159
+      payload:{
+        body: 271828 bytes of zeros
+      }
+    }
+    ```
+
+ 1. Client calls UnaryCall with the *compressed* message:
+
+    ```
+    {
+      expect_compressed: true
+      response_size: 314159
+      payload:{
+        body: 271828 bytes of zeros
+      }
+    }
+    ```
+
+ 1. Client calls UnaryCall with the *uncompressed* message:
```

Comment:
I use "the" because I'm showing the precise message just below, not an undetermined one.